### PR TITLE
Events overview page

### PR DESCRIPTION
**Pull request by christiankn**
Changes to schema:
- Add datetime to simpleevent

Work on #77

### CODE REVIEW

**Comment by christiankn** [comment]
Need to tweak alignment across rows later. It's at a point where it's open for feedback, note that the design has changed since the image used in the issue. 

Small changes to schema, and added more info to the event-cards since we won't implement single-event-page.

desktop
mobile
desktop-single

File: sanity/schemas/event-overview.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+			name: "image",
+			type: "image",
+			options: {
+				hotspot: true
+			},
+			validate: Rule => Rule.required()
+		}
+	],
+	preview: {
+		prepare: () => ({ title: "Front Page" })
```
Comment (boyeborg) [inline sanity/schemas/event-overview.js:36]: This should say "Event Overview"

File: web/src/pages/event-overview.tsx [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+	if (archive === undefined) return <div>Loading...</div>;
+	if (archive === null) return <div>404 - Not found</div>;
+
+	return (
+		<>
+			<Hero
+				angleDirection="<"
+				anglePosition="after"
+				height="50vh"
+				color={[theme.color.main.purple, theme.color.main.pink]}
```
Comment (boyeborg) [inline web/src/pages/event-overview.tsx:204]: On big screens, the text are not centred:

Screenshot 2020-05-29 at 12 33 46

I think adding `textPosition="center"` to the hero props will fix it.

**Comment by boyeborg** [comment]
Also, the price is used as the title when previewing "simple events" in sanity 😂 I think I forgot to add preview-struff. You can probably just copy what's used on the articles.

Screenshot 2020-05-29 at 12 33 18

**Pull request merged**